setup.cfg

```diff
@@ -4,3 +4,6 @@ exclude =
 	venv*
 	docs
 	build
+per-file-ignores =
+    # Need signal handler before imports
+    src/ape/__init__.py: E402
```

Comment:
why not `# noqa: E402` in the file?

Reply:
I could put `# noqa: E402` after every import in the file, but that is effectively that same as this.

I couldn't figure out a way to ignore the entire file using a comment within the file, unless you know?

Reply:
ah, I misunderstood. this is fine lol

I think there's a way to do it at the top of the file, but I don't remember how.

Reply:
Wonder if you could just get `isort` to ignore the ordering of the import :thinking:

Reply:
I think `isort` is actually okay with this. `flake8` i the one with the problem.